### PR TITLE
[Feature] Support notification platform "none"

### PR DESCRIPTION
**Pull request by mejiagarcia**
## Ticket
- [22999]

## Description
- Null PushToken supported.
- Example updated to send null token when the sample is running on a simulator

## Commit message
-  feat: Support notification platform "none" to allow not sending push token. Factors with notification platform "none" will not receive push notifications for challenges and polling should be implemented to get pending challenges

## Testing
- [x] Added unit tests
- [x] Ran unit tests successfully
- [x] Added documentation for public APIs and/or Wiki

### CODE REVIEW

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift [hunk 1]
```diff
@@ -40,8 +56,13 @@ class PushFactory {
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String,
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:59]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0c8d089e020078c52d469dcc5b1c3a5eb36c4ee1/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L59"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 43 lines<br />
<code>function_body_length</code> <code>PushFactory.swift:59</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift [hunk 2]
```diff
@@ -25,26 +25,25 @@ public protocol FactorPayload {
   var friendlyName: String { get }
   ///The unique SID identifier of the Service.
   var serviceSid: String { get }
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
-  ///because the systems that will process this attribute assume it is not directly identifying information.
+  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information) because the systems that will process this attribute assume it is not directly identifying information.
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:28]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0c8d089e020078c52d469dcc5b1c3a5eb36c4ee1/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L28"></span>Line should be 195 characters or less: currently 206 characters<br />
<code>line_length</code> <code>FactorPayload.swift:28</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift [hunk 3]
```diff
@@ -55,10 +54,10 @@ public struct PushFactorPayload: FactorPayload {
     - serviceSid: The unique SID identifier of the Service.
     - identity: Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
                 because the systems that will process this attribute assume it is not directly identifying information.
-    - pushToken: Registration token generated by APNS when registering for remote notifications.
+    - pushToken: (Optional) Registration token generated by the FCM SDK on initial startup of your app. Sending null or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:57]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0c8d089e020078c52d469dcc5b1c3a5eb36c4ee1/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L57"></span>Line should be 195 characters or less: currently 207 characters<br />
<code>line_length</code> <code>FactorPayload.swift:57</code></td>
    </tr>
  </tbody>
</table>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.86%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift [hunk 4]
```diff
@@ -20,11 +20,27 @@
...
 
 protocol PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
-  func verifyFactor(withSid sid: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
-  func updateFactor(withSid sid: String, withPushToken pushToken: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
-  func deleteFactor(withSid sid: String, success: @escaping EmptySuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
+  func createFactor(withAccessToken accessToken: String,
+                    friendlyName: String,
+                    pushToken: String?,
```
Comment (yafuquen) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:25]: As this is an internal class, we could move this to be the latest parameter before the success and failure block, wdyt?

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift [hunk 5]
```diff
@@ -25,26 +25,25 @@ public protocol FactorPayload {
...
   public let serviceSid: String
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
+  /// Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
   ///because the systems that will process this attribute assume it is not directly identifying information.
   public let identity: String
   ///Type of the factor, push by default.
   public let factorType: FactorType = .push
-  ///Registration token generated by APNS when registering for remote notifications.
-  public let pushToken: String
+  ///(Optional) Registration token generated by the FCM SDK on initial startup of your app. Sending null or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (yafuquen) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:45]: Please keep "...by APNS when registering for remote notifications." and just add the "Sending null or empty..."

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift (hunk 3 above)
Comment (yafuquen) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:57]: Same as above

File: TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift [hunk 6]
```diff
@@ -29,16 +29,16 @@ public protocol UpdateFactorPayload {
 public struct UpdatePushFactorPayload: UpdateFactorPayload {
   ///Factor Sid
   public let sid: String
-  ///Registration token generated by APNS when registering for remote notifications
-  public let pushToken: String
+  ///(Optional) Registration token generated by the FCM SDK on initial startup of your app. Sending null or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (yafuquen) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift:32]: Same as FactorPayload comment

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift (hunk 5 above)
Comment (yafuquen) [reply TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:45]: And we should replace null by nil, right?

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift (hunk 4 above)
Comment (mejiagarcia) [reply TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift]: Sounds good! I just did it on the last pushed commit

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift (hunk 5 above)
Comment (mejiagarcia) [reply TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift]: Sounds good! I just did it on the last pushed commit

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift [hunk 7]
```diff
@@ -25,26 +25,25 @@ public protocol FactorPayload {
...
   public let serviceSid: String
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
+  /// Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
   ///because the systems that will process this attribute assume it is not directly identifying information.
   public let identity: String
   ///Type of the factor, push by default.
   public let factorType: FactorType = .push
-  ///Registration token generated by APNS when registering for remote notifications.
-  public let pushToken: String
+  ///(Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:45]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/012ddda094c6c1e8c6d0425188ef9a322b4ad2d7/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L45"></span>Line should be 195 characters or less: currently 198 characters<br />
<code>line_length</code> <code>FactorPayload.swift:45</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift [hunk 8]
```diff
@@ -55,10 +54,10 @@ public struct PushFactorPayload: FactorPayload {
     - serviceSid: The unique SID identifier of the Service.
     - identity: Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
                 because the systems that will process this attribute assume it is not directly identifying information.
-    - pushToken: Registration token generated by APNS when registering for remote notifications.
+    - pushToken: (Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift:57]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/012ddda094c6c1e8c6d0425188ef9a322b4ad2d7/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L57"></span>Line should be 195 characters or less: currently 210 characters<br />
<code>line_length</code> <code>FactorPayload.swift:57</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift [hunk 9]
```diff
@@ -29,16 +29,16 @@ public protocol UpdateFactorPayload {
 public struct UpdatePushFactorPayload: UpdateFactorPayload {
   ///Factor Sid
   public let sid: String
-  ///Registration token generated by APNS when registering for remote notifications
-  public let pushToken: String
+  ///(Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift:32]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/012ddda094c6c1e8c6d0425188ef9a322b4ad2d7/TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift#L32"></span>Line should be 195 characters or less: currently 198 characters<br />
<code>line_length</code> <code>UpdateFactorPayload.swift:32</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift [hunk 10]
```diff
@@ -29,16 +29,16 @@ public protocol UpdateFactorPayload {
...
+  public let pushToken: String?
   
   /**
   Creates an **UpdatePushFactorPayload** with the given parameters
   - Parameters:
     - sid: Factor Sid
-    - pushToken: Registration token generated by APNS when registering for remote notifications
+    - pushToken: (Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift:39]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/012ddda094c6c1e8c6d0425188ef9a322b4ad2d7/TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift#L39"></span>Line should be 195 characters or less: currently 210 characters<br />
<code>line_length</code> <code>UpdateFactorPayload.swift:39</code></td>
    </tr>
  </tbody>
</table>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.86%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.86%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.86%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by SergioFierro — APPROVED** [review]

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.86%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift [hunk 11]
```diff
@@ -40,8 +56,7 @@ class PushFactory {
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String, identity: String, pushToken: String?, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:59]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/30a9bd66eedd457fd3739d00ec0357e64b7f8b99/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L59"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 43 lines<br />
<code>function_body_length</code> <code>PushFactory.swift:59</code></td>
    </tr>
  </tbody>
</table>

Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:59]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/30a9bd66eedd457fd3739d00ec0357e64b7f8b99/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L59"></span>Line should be 195 characters or less: currently 220 characters<br />
<code>line_length</code> <code>PushFactory.swift:59</code></td>
    </tr>
  </tbody>
</table>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.16%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.86%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift [hunk 12]
```diff
@@ -40,23 +56,20 @@ class PushFactory {
...
-      let payload = CreateFactorPayload(
-        friendlyName: friendlyName,
-        type: .push,
-        serviceSid: serviceSid,
-        identity: identity,
-        config: config,
-        binding: binding,
-        accessToken: accessToken
-      )
+      let payload = CreateFactorPayload(friendlyName: friendlyName,type: .push,
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:68]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/7971036277c0571b3d83d953e1f5c0855e226bd3/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L68"></span>There should be no space before and one after any comma.<br />
<code>comma</code> <code>PushFactory.swift:68</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift [hunk 13]
```diff
@@ -40,23 +56,20 @@ class PushFactory {
...
+                                        serviceSid: serviceSid,identity: identity,
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:69]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/7971036277c0571b3d83d953e1f5c0855e226bd3/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L69"></span>There should be no space before and one after any comma.<br />
<code>comma</code> <code>PushFactory.swift:69</code></td>
    </tr>
  </tbody>
</table>

File: TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift [hunk 14]
```diff
@@ -40,23 +56,20 @@ class PushFactory {
...
+                                        config: config,binding: binding,
```
Comment (AuthyApps) [inline TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift:70]: <table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/7971036277c0571b3d83d953e1f5c0855e226bd3/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L70"></span>There should be no space before and one after any comma.<br />
<code>comma</code> <code>PushFactory.swift:70</code></td>
    </tr>
  </tbody>
</table>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.15%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.83%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.15%`**
File | Coverage
-----|-----
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`98.83%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift | **`100.00%`**
TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by yafuquen — APPROVED** [review]

**Review by AlejandroOrozco — APPROVED** [review]
LG 👍

**Pull request merged**